odoo/addons/base/models/res_partner.py

```diff
@@ -360,11 +360,6 @@ def _onchange_state(self):
         if self.state_id.country_id:
             self.country_id = self.state_id.country_id
 
-    @api.onchange('email')
-    def onchange_email(self):
-        if not self.image_1920 and self._context.get('gravatar_image') and self.email:
-            self.image_1920 = self._get_gravatar_image(self.email)
-
```

Comment:
Is it a good idea to remove this actually?

I think the value from gravatar (if it exists) is still better than our placeholder.

Reply:
But this is setting an image for the user. It's completely the opposite of the goal of this task, since our avatars won't be used

Reply:
> Is it a good idea to remove this actually?
> 
> I think the value from gravatar (if it exists) is still better than our placeholder.

Probably not. Reomving gravatar is the right thing to do :)

Reply:
I just checked with NGR, as this was not thought about and not the goal of the task, he prefers that we keep it here, at least in the context of the current PR.
Separate feedback can be done to determine its fate.

Reply:
Well what is the point of keeping calls to an external site when we now have our own avatar generator ? We even had to add hacks to avoid calls to gravatar adding latency a bit everywhere.

Reply:
I bet PME don't use gravatar. This is clearly out of scope for odoo.

Reply:
I agree gravatar is more for forum and shit. Have you check alternative algorithms like https://boringavatars.com (open source) which are a bit more pretty than just one letter?